src/static/clientConfig.test.ts

```diff
@@ -67,41 +67,31 @@ describe('ClientConfig constructor', () => {
 describe('ClientConfig defaults', () => {
   describe('transform request', () => {
     const transform = ClientConfig.defaults.transformRequest;
-    let headers: {[key: string]: string};
 
-    beforeEach(() => {
-      headers = {};
+    test('converts to JSON for Content-Type application/json', () => {
+      expect(
+        transform({json: true}, {'Content-Type': 'application/json'})
+      ).toEqual('{"json":true}');
     });
 
-    test('does not modify body or headers for primitive values', () => {
-      expect(transform(null, headers)).toBeNull();
-      expect(headers).toEqual({});
-
-      expect(transform(undefined, headers)).toBeUndefined();
-      expect(headers).toEqual({});
-
-      expect(transform('a string', headers)).toBe('a string');
-      expect(headers).toEqual({});
-    });
-
-    test('does not modify body or headers for non-plain objects', () => {
-      const blob = new Blob();
-      expect(transform(blob, headers)).toBe(blob);
-      expect(headers).toEqual({});
-    });
-
-    test('converts plain objects to JSON and sets Content-Type header', () => {
-      expect(transform({plain: 'object'}, headers)).toBe('{"plain":"object"}');
-      expect(headers).toEqual({'Content-Type': 'application/json'});
-
-      headers = {};
-      expect(transform(Object.create(null), headers)).toBe('{}');
-      expect(headers).toEqual({'Content-Type': 'application/json'});
```

Comment:
Aren't these test cases still relevant?

Reply:
They are not. The implementation has changed and no longer does this.